src/picotorrent/ui/dialogs/preferencesgeneralpage.cpp

```diff
@@ -145,6 +150,16 @@ PreferencesGeneralPage::PreferencesGeneralPage(wxWindow* parent, std::shared_ptr
         }
     }
 
+    for (auto &theme : pt::UI::Theming::GetInstance().GetThemes())
```

Comment:
Given that there's only three options that shouldn't really change, we can simplify this to just adding those three static options.